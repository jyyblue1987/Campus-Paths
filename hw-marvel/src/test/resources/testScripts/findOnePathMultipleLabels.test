# checks if the output is correct for paths with multiple labels

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n1 n2 a
AddEdge g n1 n2 b
AddEdge g n2 n3 c
AddEdge g n2 n3 d
AddEdge g n3 n4 e
AddEdge g n3 n4 f

FindPath g n1 n4