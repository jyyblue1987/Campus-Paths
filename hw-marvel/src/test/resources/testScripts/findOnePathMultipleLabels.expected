# checks if the output is correct for paths with multiple labels

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge a from n1 to n2 in g
added edge b from n1 to n2 in g
added edge c from n2 to n3 in g
added edge d from n2 to n3 in g
added edge e from n3 to n4 in g
added edge f from n3 to n4 in g

path from n1 to n4:
n1 to n2 via a
n2 to n3 via c
n3 to n4 via e